### PR TITLE
chore: Add release notes for Bloop v1.5.16

### PR DESCRIPTION
**Pull request by tgodzik**

### CODE REVIEW

**Review by adpi2 — COMMENTED** [review]
Is this the Bloop release for the next Metals release?

I am asking because I still want to include some minor changes in scala-debug-adapter and release.

**Comment by tgodzik** [comment]
> Is this the Bloop release for the next Metals release?
> 
> I am asking because I still want to include some minor changes in scala-debug-adapter and release.

It's mostly to get native support out, we can do another release before Metals release

File: notes/v1.5.16.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# bloop `v1.5.16`
+
+Bloop v1.5.16 is a bugfix release. Main changes involve adding support for Scala
```
Comment (adpi2) [inline notes/v1.5.16.md:3]: I would not say its a bugfix release if it adds support for Scala Native 0.5.x.

```suggestion
Main changes involve adding support for Scala
```

File: notes/v1.5.16.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+Native 0.5.x and limiting amount of compile notifications being sent when noop
```
Comment (adpi2) [inline notes/v1.5.16.md:4]:
```suggestion
Native 0.5., adding support for hot-code reload and limiting amount of compile notifications being sent when noop
```

:)

Comment (tgodzik) [reply notes/v1.5.16.md]: And done

**Review by adpi2 — APPROVED** [review]

**Pull request merged**